### PR TITLE
Ehr transformer example

### PR DESCRIPTION
**Pull request by ellabarkan**
Added unitest for ehr_transormer. Following that, some minor changes were done to the files of the example

### CODE REVIEW

**Review by mosheraboh — COMMENTED** [review]
Thanks Ella,
The code looks good.
But it doesn't seem that CICD pipeline runs this test.
Any chance that you didn't add CINC_TEST_DATA_PATH to envs.sh?

File: examples/fuse_examples/tests/test_ehr_transformer.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        main_train(cfg)
+
+    except:
+        raise Exception("Training EHR Transformer Failed.")
+
+
+@unittest.skipIf(
+    "CINC_TEST_DATA_PATH" not in os.environ, "define environment variable 'CINC_TEST_DATA_PATH' to run this test"
+)
+class EHRTransformerTestCase(unittest.TestCase):
```
Comment (mosheraboh) [inline examples/fuse_examples/tests/test_ehr_transformer.py:75]: You should store all the output in the temp folder for cases that few PRs testes in parallel.

Comment (ellabarkan) [reply examples/fuse_examples/tests/test_ehr_transformer.py:71]: Fixed and pushed

**Review by mosheraboh — APPROVED** [review]
Looks good
You can go ahead and merge it

**Pull request merged**